ROMFS/px4fmu_common/init.d-posix/airframes/4008_gz_advanced_plane

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+#
+# @name Advanced Plane SITL
+#
+
+. ${R}etc/init.d/rc.fw_defaults
+
+PX4_SIMULATOR=${PX4_SIMULATOR:=gz}
+PX4_GZ_WORLD=${PX4_GZ_WORLD:=default}
+PX4_SIM_MODEL=${PX4_SIM_MODEL:=Advanced_Plane}
+
+param set-default SIM_GZ_EN 1
+
+param set-default SENS_EN_GPSSIM 1
+param set-default SENS_EN_BAROSIM 0
+param set-default SENS_EN_MAGSIM 1
+param set-default SENS_EN_ARSPDSIM 1
```

Comment:
You have an airspeed sensor attached in gazebo, so you don't need a simulated airspeed sensor

Reply:
![Screenshot from 2023-10-03 09-40-23](https://github.com/PX4/PX4-Autopilot/assets/80588263/340c9ceb-16a6-4e95-878b-ff84e5cdcedf)

The simulation complains if the airspeed sensor is not defined in the airframe file

Reply:
Ah, it was the air pressure sensor -> can we use the gazebo simulated airspeed sensor?

Reply:
Hmm maybe? I tried placing the air speed sensor in the sdf file and took out the ARSPDSIM from the airframe but it then again complains about the invalid airspeed. However, maybe I just defined it wrong in the sdf file? Where can I find the documentation for the sensor (not the class reference but the description of what variables can be written into the sdf file).

Reply:
Lets address this in a followup